server-kotlin/src/test/kotlin/digital/cesko/internet_stream/InternetStreamServiceTest.kt

```diff
@@ -19,14 +19,15 @@ class InternetStreamServiceTest : AbstractSpringDatabaseTest() {
     fun fetchBudgets() {
         internetStreamService.fetchData()
         transaction {
-            val result = Payments.select { Payments.year eq 2020 }.first()
+            val result = Payments.select { Payments.year eq 2020 }.toList()
```

Comment:
I do not understand this change

Reply:
I insert more data and retrieve 2 rows from db.